docs/becoming-a-maintainer/translations/es/como-configurar-tu-proyecto.md

```diff
@@ -0,0 +1,208 @@
+# Cómo configurar tu proyecto Open Source
```

Comment:
Consider using the following instead 

"Cómo configurar tu proyecto de Open Source"